translator/translate/otel/receiver/otlp/translator.go

```diff
@@ -77,26 +85,48 @@ func (t *translator) ID() component.ID {
 
 func (t *translator) Translate(conf *confmap.Conf) (component.Config, error) {
 	cfg := t.factory.CreateDefaultConfig().(*otlpreceiver.Config)
-
-	// TODO: Should follow pattern done in awsemf and awsexray exporter translations (i.e should be integrated with standard otlp translation)
+	// init default configuration
+	configKeyExt := common.OtlpKey
+	cfg.GRPC.NetAddr.Endpoint = defaultGrpcEndpoint
+	cfg.HTTP.Endpoint = defaultHttpEndpoint
 	if t.name == common.AppSignals {
-		return common.GetYamlFileToYamlConfig(cfg, appSignalsConfig)
+		configKeyExt = common.AppSignals
```

Comment:
Is `appSignalsConfig` and `appsignals_config.yaml` no longer being used?

Reply:
yes, I'll remove it